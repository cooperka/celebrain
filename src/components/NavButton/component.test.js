// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import Button from './component';

it('renders without border', () => {
  const tree = renderer.create(<Button linkTo="/test">Click me</Button>);
  expect(tree.toJSON()).toMatchSnapshot();
});

it('renders with border', () => {
  const tree = renderer.create(<Button bordered linkTo="/test">Click me</Button>);
  expect(tree.toJSON()).toMatchSnapshot();
});
