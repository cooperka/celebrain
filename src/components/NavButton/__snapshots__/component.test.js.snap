// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with border 1`] = `
<div
  className="NavButton bordered"
>
  <Link
    className="material waves-effect waves-teal btn-flat"
    to="/test"
  >
    <div
      className="link-text"
    >
      Click me
    </div>
  </Link>
</div>
`;

exports[`renders without border 1`] = `
<div
  className="NavButton "
>
  <Link
    className="material waves-effect waves-teal btn-flat"
    to="/test"
  >
    <div
      className="link-text"
    >
      Click me
    </div>
  </Link>
</div>
`;
